src/libraries/System.Private.Xml/tests/XmlSerializer/XmlSerializerTests.RuntimeOnly.cs

```diff
@@ -397,18 +397,51 @@ public static void Xml_EnumerableGenericRoot()
     [Fact]
     public static void Xml_CollectionRoot()
     {
-        MyCollection x = new MyCollection('a', 45);
+        DateTime now = new DateTime(2022, 9, 30, 9, 4, 15, DateTimeKind.Utc);
+        DateTimeOffset dtoNow = now.AddDays(1);
+        TimeSpan ts = new TimeSpan(1, 2, 3, 4, 5);
+        MyCollection x = new MyCollection('a', 45,
+            123.45m, now, ts, dtoNow, (short)55, 2345324L, (sbyte)11, (ushort)34, (uint)4564, (ulong)456734767,
+            new byte[]{ 33, 44, 55}, (byte)67);
         MyCollection y = SerializeAndDeserialize<MyCollection>(x,
-@"<?xml version=""1.0""?>
+@"<?xml version=""1.0"" encoding=""utf-8""?>
```

Comment:
any particular reason for changing tests? can you add new tests instead and leave existing one as is so that we can be sure there is no regression?